manifests/assertions.json

```diff
@@ -86,14 +86,6 @@
         }]
       }
     },
-    "OB3GLOAssertBulkNumberOfPayments": {
-      "expect": {
-        "matches": [{
-          "JSON": "Data.StandingOrder.#.NumberOfPayments",
-          "detail": "Expected NumberOfPayments (array) to be present."
-        }]
-      }
-    },
     "OB3GLOAssertNoNumberOfPayments": {
       "expect": {
```

Comment:
These assertions were based on the v3 schema where you can only have NumberOfPayments or FinalPaymentDateTime: Either the NumberOfPayments or FinalPaymentDateTime must be specified (not both) if the domestic standing order is not open ended

In v4 this has been replaced with this rule: Either the CountPerPeriod or FinalPaymentDateTime must be specified (not both) if the domestic standing order is not open ended.

The assertion(s) checking for NumberOfPayments should be updated in v4 to look for CountPerPeriod.  See the [UML diagram](https://openbankinguk.github.io/read-write-api-site3/v4.0/resources-and-data-models/pisp/domestic-standing-order-consents.html#uml-diagram) for the current location of these fields

Reply:
As discussed on teams. This is a PR to remove unused assertions. These assertions are not used in v3 nor in v4. All relevant v4 assertions will come on future PRs. Keeping track of all assertions being updated in the FCS v4 spreadsheet https://openservices-my.sharepoint.com/:x:/r/personal/peter_stanley_openbanking_org_uk/_layouts/15/Doc.aspx?sourcedoc=%7B28D9BAB8-FC50-49DF-9928-3B6128C4457C%7D&file=FCS%20Test%20List.xlsx&action=default&mobileredirect=true